dirsrvtests/tests/suites/password/pbkdf2_upgrade_plugin_test.py

```diff
@@ -35,18 +35,18 @@ def test_pbkdf2_upgrade(topology_st):
 
     """
     # Remove the pbkdf2 plugin config
-    p1 = PBKDF2Plugin(topology_st.standalone)
+    p1 = PBKDF2SHA512Plugin(topology_st.standalone)
```

Comment:
Should we really remove the test about the legacy plugin ?
IMHO it should be better to keep it and add a similar test for the new plugin

Reply:
Old test was about `PBKDF2-SHA256` plugin (as `PBKDF2Plugin` was about it). Now, it's `PBKDF2SHA256Plugin`. But yeah, I agree, better not to change it in this PR, even if we want to test for `PBKDF2-SHA512` here. I'll change it!